Success: An exception was thrown.
